sql/hive/src/main/java/org/apache/hadoop/hive/ql/io/orc/SparkOrcNewRecordReader.java

```diff
@@ -42,8 +41,11 @@
 
   public SparkOrcNewRecordReader(Reader file, Configuration conf,
       long offset, long length) throws IOException {
-    List<OrcProto.Type> types = file.getTypes();
-    numColumns = (types.size() == 0) ? 0 : types.get(0).getSubtypesCount();
+    if (file.getTypes().isEmpty()) {
```

Comment:
The compiler prints warnings for the code:
```
/Users/maxim/proj/orc-eliminate-warning/sql/hive/src/main/java/org/apache/hadoop/hive/ql/io/orc/SparkOrcNewRecordReader.java
Warning:(44, 13) java: getTypes() in org.apache.orc.Reader has been deprecated
Warning:(47, 24) java: getTypes() in org.apache.orc.Reader has been deprecated
```
I am going to rewrite it slightly. I am wondering how this was tested?

Reply:
Why do you ask here? `file.getTypes()` seems to be used before this PR.
Did you see the removed line 45?

Reply:
> Why do you ask here? 

The PR changed this code, and probably, it was tested somehow. PR's description mentions `manual tests`. That's why I asked how. I have found a few tests like `org.apache.spark.sql.hive.orc.HiveOrcQuerySuite`. Will it be enough or not.